scripts/st2bootstrap-deb.sh

```diff
@@ -119,15 +126,22 @@ setup_args() {
     sudo apt-get update > /dev/null 2>/dev/null
     # check if python3.6 is available
     if (! apt-cache show python3.6 2> /dev/null | grep 'Package:' > /dev/null); then
-      echo ""
-      echo "WARNING!"
-      echo "The python3.6 package is a required dependency for the StackStorm st2 package but that is not installable from any of the default Ubuntu 16.04 repositories."
-      echo "We recommend switching to Ubuntu 18.04 LTS (Bionic) as a base OS. Support for Ubuntu 16.04 will be removed with future StackStorm versions."
-      echo ""
-      echo "Alternatively we'll try to add python3.6 from the 3rd party 'deadsnakes' repository: https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa."
-      echo "By continuing you are aware of the support and security risks associated with using unofficial 3rd party PPA repository, and you understand that StackStorm does NOT provide ANY support for python3.6 packages on Ubuntu 16.04."
-      echo ""
-      read -p "Press [y] to continue or [n] to cancel adding it: " choice
+      if [[ "$U16_ADD_INSECURE_PY3_PPA" = "1" ]]; then
+        choice=y
+      else
+        echo ""
+        echo "WARNING!"
+        echo "The python3.6 package is a required dependency for the StackStorm st2 package but that is not installable from any of the default Ubuntu 16.04 repositories."
+        echo "We recommend switching to Ubuntu 18.04 LTS (Bionic) as a base OS. Support for Ubuntu 16.04 will be removed with future StackStorm versions."
+        echo ""
+        echo "Alternatively we'll try to add python3.6 from the 3rd party 'deadsnakes' repository: https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa."
+        echo "By continuing you are aware of the support and security risks associated with using unofficial 3rd party PPA repository, and you understand that StackStorm does NOT provide ANY support for python3.6 packages on Ubuntu 16.04."
+        echo ""
+        echo "To bypass this check in future, you can provide the following flag: --u16-add-insecure-py3-ppa"
+        echo ""
+        echo "Press [y] to continue or [n] to cancel adding it: "
+        read choice
```

Comment:
FYI in a sub-shell environment `read -p "Message" choice` won't show the Message and so that was moved out.